Projects/Presentation/Sources/Home/EmployStatus/EmployStatusViewController.swift

```diff
@@ -89,6 +91,19 @@ public final class EmployStatusViewController: BaseViewController<EmployStatusVi
                 self?.chartView.setChartData(model: info)
             }
             .disposed(by: disposeBag)
+
+        [
+            (classButton1, 1),
+            (classButton2, 2),
+            (classButton3, 3),
+            (classButton4, 4)
+        ].forEach { button, classNumber in
+            button.rx.tap
+                .map { classNumber }
+                .bind(to: classButtonTapped)
+                .disposed(by: disposeBag)
+        }
```

Comment:
```suggestion
        [
            classButton1,
            classButton2,
            classButton3,
            classButton4
        ]
        .enumerated()
        .forEach { classNumber, button in
            button.rx.tap
                .map { classNumber + 1 }
                .bind(to: classButtonTapped)
                .disposed(by: disposeBag)
        }
```
반이 늘어나는 경우는 없겠지만 직접 숫자를 적는 것 보단 enumerated()를 사용해 index를 가져오는 방법도 있어요!
또 classButton이 UIButton 이기에 set하는 부분에서 tag로 classNumber를 심어주어 tap할 경우 tag로 mapping 할 수도 있을거 같아요

Reply:
@HongSJae